vendors/llama.go

```diff
@@ -0,0 +1,44 @@
+package vendors
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/flair-sdk/erpc/common"
+)
+
+type LlamaVendor struct {
+	common.Vendor
+}
+
+func CreateLlamaVendor() common.Vendor {
+	return &DrpcVendor{}
+}
+
+func (v *LlamaVendor) Name() string {
+	return "llama"
+}
+
+func (v *LlamaVendor) GetVendorSpecificErrorIfAny(resp *http.Response, jrr interface{}) error {
+	bodyMap, ok := jrr.(*common.JsonRpcResponse)
+	if !ok {
+		return nil
+	}
+
+	err := bodyMap.Error
+	code := err.OriginalCode()
+	msg := err.Message
+
+	if code == 0 && strings.Contains(msg, "error code: 1015") {
```

Comment:
Not sure if by default non-json responses (i.e. Llama's rate limits) will be properly hydrated into the error object. 

A unit test can show how it works. Maybe to have a fallback behavior when body is non-parsable to use the body as-is in the "message" field. That way this logic here will work.